src/Serilog.Extras.Timing/Extras/Timing/CounterMeasure.cs

```diff
@@ -46,8 +49,10 @@ public CounterMeasure(ILogger logger, string name, string counts, LogEventLevel
             _value = new AtomicLong();
         }
 
-
-        public void Increment()
+		/// <summary>
+		/// Increments the counter.
+		/// </summary>
+        public virtual void Increment()
```

Comment:
Haven't read ahead; but to support higher perf scenarios the original would have been getting close to an order of magnitude faster - worth considering?